### DISCUSSION TITLE
Better control for File loading

### DISCUSSION THREAD

**Discussion by JM-Bader** [MannLabs/alphadia#111] [Ideas]
make it possible to load folders that contain timsTOF subfodlders. 
it would be informative to show the file path of the master folder, not the raw file subfolders. 
eg. I have 50 master folders with  a 96well plate each

**Comment by JM-Bader** [comment]
Please also give option to remove loaded raw files or the master folders. 
Master folders could be similar to the DIA-NN command line for taht, e.g. --dir ....

**Reply by mschwoer** [reply]
this can be done by using the `--regex` feature of the CLI

**Comment by mschwoer** [comment]
answered
